logpuzzle.py

```diff
@@ -40,14 +62,26 @@ def download_images(img_urls, dest_dir):
     with an img tag to show each local image file.
     Creates the directory if necessary.
     """
+
+    path = os.getcwd()
+    os.mkdir(path+"/"+dest_dir)
+
     # +++your code here+++
-    pass
+    f = open(path+"/"+dest_dir+"/index.html", "w")
+    f.write("<html>\n<body>\n")
+    for i, url in enumerate(img_urls):
+        urllib.urlretrieve(url, path+"/"+dest_dir+"/"+"img"+str(i)+".jpg")
```

Comment:
I like to compose url and file path strings outside of function calls.  That way I can see their final values as parameters into a function call when stepping in the debugger.